lib/rucio/api/did.py

```diff
@@ -469,17 +469,18 @@ def get_metadata(scope, name, plugin='DID_COLUMN', vo='def'):
     return api_update_return_dict(d)
 
 
-def get_metadata_bulk(dids, vo='def', session=None):
+def get_metadata_bulk(dids, inherit=False, vo='def'):
     """
     Get metadata for a list of dids
-    :param dids: A list of dids.
-    :param session: The database session in use.
+    :param dids:               A list of dids.
+    :param inherit:            A boolean. If set to true, the metadata of the parent are concatenated.
+    :param vo:                 The VO to act on.
     """
 
     validate_schema(name='dids', obj=dids, vo=vo)
     for entry in dids:
         entry['scope'] = InternalScope(entry['scope'], vo=vo)
-    meta = did.get_metadata_bulk(dids)
+    meta = did.get_metadata_bulk(dids, inherit=inherit)
```

Comment:
is this missing the vo selection?

Reply:
Nope, there is no vo in the signature of the core method. The vo filtering is done when the `InternalScope` is extracted at the previous line

Reply:
The only tests that fail are the webpy ones, but webpy is not available anymore... I don't understand